projects/valtimo/task/src/lib/components/task-detail-modal/task-detail-modal.component.ts

```diff
@@ -158,12 +164,12 @@ export class TaskDetailModalComponent implements AfterViewInit, OnDestroy {
 
     //only load from formlink when process link failed for backwards compatibility
     if (!this.taskProcessLinkType$.getValue()) {
-      this.modal.show();
+      this.modal.open = true;
```

Comment:
try this.modal = {...this.modal, open: true} 

Otherwise, change detection might fail in the future